index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Site</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+  
+</head>
+<body>
+  <header class="head">
+    <nav>
+      <h3 class="logo">Portfolio</h3>
+      <i class="fa fa-bars"></i>   
+    </nav>
+  </header>
+  <main>
+    <div class="container">
+      <h1>
+      I'm Uchechi<br>
```

Comment:
- [ ] Consider using semantic tags for better code readability, user experience, and greater accessibility. Kindly it's better to use a  semantic tag like `section` instead of a `div`. :eagle:

Reply:
Dang